src/main/python/systemds/matrix/data_gen.py

```diff
@@ -102,3 +105,70 @@ def rand(sds_context: SystemDSContext, rows: int, cols: int,
         named_input_nodes['seed'] = seed
 
     return OperationNode(sds_context, 'rand', [], named_input_nodes=named_input_nodes)
+
+
+def rev(sds_context: 'SystemDSContext', X: Matrix) -> 'OperationNode':
```

Comment:
In the Methods, i would appreciate if the syntax is:

` def rev(sds_context: SystemDSContext, X: Matrix) -> OperationNode:`

Since it allows the system to complain earlier about missing imports and such not resulting in a potential runtime error.